include/LLVMSPIRVExtensions.inc

```diff
@@ -18,3 +18,4 @@ EXT(SPV_INTEL_io_pipes)
 EXT(SPV_INTEL_inline_assembly)
 EXT(SPV_INTEL_arbitrary_precision_integers)
 EXT(SPV_INTEL_optimization_hints)
+EXT(SPV_KHR_float_controls)
```

Comment:
Should we move this to the top, so that the SPV_KHR extensions are together?

Reply:
Fixed